Sources/APIProvider.swift

```diff
@@ -75,7 +75,12 @@ public final class APIProvider {
     /// Contains a JSON Decoder which can be reused.
     static let jsonDecoder: JSONDecoder = {
         let decoder = JSONDecoder()
+        
         let formatter = DateFormatter()
+        formatter.locale = Locale(identifier: "en_US_POSIX")
+        formatter.timeZone = TimeZone(secondsFromGMT: 0)
+        formatter.calendar = Calendar(identifier: .iso8601)
```

Comment:
Wouldn't it be enough to set this calendar? Do we really need the timezone and locale setting?

Reply:
If we set only the `calendar` parameter, so `locale` and `timeZone` will stay by default (according to the user iOS Settings). And for ISO8601-like DateFormatter it's expected `en_US_POSIX` locale and `GMT` timeZone. I think it would be more stable that way.

Reply:
Alright, makes sense! Thanks for the detailed explanation 🙂